phylogenetic/scripts/map_accessions.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python3
+"""
+Map INSDC accessions to PPX accessions using metadata.
```

Comment:
Why not update the accession list files to use Pathoplexus accessions? That was my thinking in https://github.com/nextstrain/WNV/commit/34ea83efef9dbd4d3278a5339af8a1990fc60f80.